.circleci/config.yml

```diff
@@ -30,9 +30,86 @@ jobs:
           name: Update CodeClimate coverage report
           command: |
               ./cc-test-reporter after-build --exit-code $?
+  build-and-upload-image:
+    docker:
+      - image: docker:20.10.14-git
+    parameters:
+      app_name:
+        type: string
+        description: Name of the application to build
```

Comment:
Is this the right description?

Reply:
In that step, the app_name parameter is declared without a value. Below, in the workflows jobs, the value to app_name  is assigned to edagames-wumpus.

In our case, the application to build is wumpus. Do you suggest changing the description to "wumpus app" or something like that?

```
workflows:
  main:
    jobs:
      - build-and-test
      - build-and-upload-image:
          app_name: edagames-wumpus
```

Reply:
Nice! I see. You have my 🟢